code/datums/diseases/advance/symptoms/stinky.dm

```diff
@@ -0,0 +1,42 @@
+/*
+//////////////////////////////////////
+Stinky Syndicate
+	Somewhat Noticable.
+	Increases resistance.
+	Increases stage speed.
+	Very transmittible.
+	High Level.
+Bonus
+	Forces a spread type of AIRBORNE
+	with extra range!
+//////////////////////////////////////
+*/
+
+/datum/symptom/stinky
+	name = "Stinky Syndicate"
+	desc = "A highly resistant virus that causes rapid secretion of body odors that spread to nearby hosts."
+	stealth = 2
+	resistance = 10
+	stage_speed = 4
+	transmittable = 4
+	level = 666
+	severity = 1
+	symptom_delay_min = 5
+	symptom_delay_max = 35
+	threshold_desc = "<b>Transmission 9:</b> Increases stink range, spreading the virus over a larger area."
+
+
+/datum/symptom/stinky/Start(datum/disease/advance/A)
+	if(!..())
+		return
+	if(A.properties["transmittable"] >= 9) //longer spread range
+		power = 2
+
+/datum/symptom/stinky/Activate(datum/disease/advance/A)
+	if(!..())
+		return
+	var/mob/living/carbon/M = A.affected_mob
+	switch(A.stage)
```

Comment:
Before this. 
if(iscarbon(M))
Checks that their mob is a human and not some freak of nature.

Reply:
Will do.

Reply:
Did I do this right?

Reply:
Apparently not, I am trying different things to see how it works now.